standalone/elf.c

```diff
@@ -57,61 +56,94 @@ gen_elf_segment(uint8_t **code, uintptr_t target, void *src, size_t len,
   byte_out(code, 0xAA);         /* STOSB */
 }
 
-int
-start_module(struct mbi *mbi, bool uncompress, uint64_t phys_max)
+int for_each_phdr(uint32_t const binary, uint8_t ** var,
```

Comment:
Taking const ints as parameters has no use. Shouldn't this function take a `struct eh const *binary` instead of a `uint32_t`?

Reply:
Given the non-obvious function prototype it would be nice to have a bit of documentation what the magic `uint8_t **` pointers are and the other `uint32_t const` (why another const int?) are.